plrust/src/tests.rs

```diff
@@ -1426,7 +1494,8 @@ pub mod pg_test {
 owo-colors = "=3.5.0"
 tokio = { version = "=1.19.2", features = ["rt", "net"]}
 plutonium = "*"
-rand = "*"
+syn = { version = "=2.0.28", default-features = false }
+rand = ["=0.8.3", { version = ">0.8.4, <0.8.6", features = ["getrandom"]}]
```

Comment:
Hmm, do we... actually want this? Is this valid in a Cargo.toml? I would think not?

Reply:
this is not a valid Cargo.toml format, but i think plrust is able to interpret that format.

https://tcdi.github.io/plrust/dependencies.html#the-format

Reply:
I... missed that in my code review after my vacation.

Reply:
Ah okay I see what's going on here at second glance. This is an array of entries for a single version, and each is actually independent. It first struck me as a bit weird but I grok the intention here now.